build/gm/gm-config.yaml

```diff
@@ -6,3 +6,5 @@ websocket:
   port: 9000
 localController:
   server: http://localhost:9100
+knowledgeBaseService:
+  server: http://localhost:9020
```

Comment:
knowledgeBaseService is run as a k8s service, we can use the serviceName/port instead of localhost

Reply:
fix it